lms/djangoapps/teams/serializers.py

```diff
@@ -0,0 +1,140 @@
+"""
+Defines serializers used by the Teams API.
```

Comment:
Nit: I've been calling it the Team API (singular) to mirror the User API, the Profile Image API etc.